tests/integration/012_environment/main.tf

```diff
@@ -55,7 +55,7 @@ resource "env0_environment" "example" {
   depends_on    = [env0_template_project_assignment.assignment]
```

Comment:
Is this environment still necessary?

Reply:
yes. tomer just added the move part [here](https://github.com/env0/terraform-provider-env0/pull/939)

Reply:
I think the original purpose of the test was to test different types / settings of environments, and here there was an attempt to kill 2 birds with one stone by testing both regular deployment and move environment. so I think it is still needed for the original purpose